### ISSUE TITLE
Worlds: controlling update sequence.

### ISSUE THREAD

**Issue by Yanrishatum** [useflashpunk/FlashPunk#124]
I'm lack of controlling update sequence in Worlds, and added some functions to control it. In perfect way - there must be something like layers for update sequences, but it's to complicated.
I'm made next functions:
- Bring entity to be updated first.
- Send entity to be updated last.
- Bring entity forward one step (updated earlier).
- Send entity backward one step (updated later).
- Bring entity to be updated before second entity.
- Send entity to be updated after second entity.
- Swap two entities in update sequence.

Source: http://pastebin.com/02LePwHE
I apologise for external site URL. I didn't want to make a fork or post all code, because it takes too many space.

**Comment by citiral** [comment]
I agree, these functions can really come in handy for some entities that need to get update priority. I'd say make a branch and pull request right away
